cmd/cogito/main.go

```diff
@@ -50,21 +49,14 @@ func mainErr(in io.Reader, out io.Writer, logOut io.Writer, args []string) error
 	})
 	log.Info(cogito.BuildInfo())
 
-	ghAPI := os.Getenv("COGITO_GITHUB_API")
-	if ghAPI != "" {
-		log.Info("endpoint override", "COGITO_GITHUB_API", ghAPI)
-	} else {
-		ghAPI = github.API
-	}
-
 	switch cmd {
 	case "check":
 		return cogito.Check(log, input, out, args[1:])
 	case "in":
 		return cogito.Get(log, input, out, args[1:])
 	case "out":
-		putter := cogito.NewPutter(ghAPI, log)
-		return cogito.Put(log, input, out, args[1:], putter)
```

Comment:
`log` variable was really unused. There is another option where we can not set the log in putter structure, but send it always as a function parameter, but that is too painful in my opinion. Too many code changes for no gain.

Reply:
> I am surprised here, because I remember we had to set the log here (see line 45) because of line 41, were we "peek" for the log level before parsing the JSON of the input.

Yes, sure. that is still the case. There is no changes to that "logic"

> Ah. Do you mean that the putter (line 58) still uses the log initialized here, while cogito.Put (line 59) currently does not log anything?

Exactly.

> Also in this case, I would be tempted to leave the log parameter to cogito.Put, because it matches the two other calls, cogito.Check and cogito.Get. One less difference.

Apparently, some linters complain about unused parameters. Our doesn't. I did the exercise and tried to make it consistent with other two calls, but its not worth it really. So we need a decision on one of the options:
- we live with unused parameter that bring the confusion (belive me it does) just to have a match with above calls
- or we remove it

Reply:
(discussed off-line)

Reply:
This change is reverted in:
https://github.com/Pix4D/cogito/pull/143/commits/95d29b29c448c536ae77895298f4d44887a83b06